### DISCUSSION TITLE
Why raw-content does not render in qmd file?

### DISCUSSION THREAD

**Discussion by LuisLauM** [quarto-dev/quarto-cli#11697] [Q&A] [answered]
### Description

Hi!

I'm trying to run a simple example in Quarto with a chunk in markdown format, but when I use the raw-content syntax nothing is displayed. I know I must be doing alo wrong, but I don't know what it is.

Windows 10 22H2
R 4.4.2
RStudio 2024.12.0 Build 467

![image](https://github.com/user-attachments/assets/1ae15955-d5dd-423e-89b3-994b887c1438)

**Comment by mcanouil** [comment]
What language is supposed to be `plain`?
What are you trying to do here?

**Comment by LuisLauM** [comment]
I expected (perhaps wrongly) that it would return a chunk in `plain`-text (verbatim). On the site of Quarto, the [raw-content section](https://quarto.org/docs/authoring/markdown-basics.html#raw-content) refers to the [`raw_attribute` section of pandoc](https://pandoc.org/MANUAL.html#extension-raw_attribute) where in turn they indicate that the list of available formats is obtained with `--list-output-formats` and one of them was `plain`, which I thought was plain text, but maybe not.

**Reply by mcanouil** [reply, marked as answer]
I still don't understand what you want to do.

"This works" is "plain" text from your screenshot (by the way, sharing code via screenshot is not the best way to do so).
"plain" is a format not a language and is described as such in the documentation you've linked.
